firmware/peripherals/moisture_sensor/Kconfig

```diff
@@ -1,3 +1,9 @@
 menu "Moisture sensor setup"
+    config DEFAULT_MIN
+    int "Default min"
+    default 356
 
+    config DEFAULT_MAX
+    int "Default max"
+    default 880
```

Comment:
The limits of the read should be constants, The previously setup of the limits could get the incorrect measurements. I think that to each resolution the code has to adapt automatically to the constants limits in that resolution, So, i see that the only parameter to modify, would be the ADC resolution